clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchFieldMapper.scala

```diff
@@ -71,6 +71,9 @@ private[dataaccess] object ElasticsearchFieldMapper
     private def isDate(tpe: Type): Boolean =
       tpe =:= typeOf[OffsetDateTime] || tpe =:= typeOf[Option[OffsetDateTime]]
 
+    private def isSymbol(tpe: Type): Boolean =
+      tpe =:= typeOf[Symbol] || tpe =:= typeOf[Option[Symbol]]
```

Comment:
We can take the chance to clean up other existing code by using this in `stringToDefinition` above, too. Same with `isDate`.

Reply:
Also, something to try: does a generic `is[A]` method work?
```scala
private def is[A](tpe: Type): Boolean =
  ... tpe checks, including check for Seq[A] ...
```
Not sure if it'll get screwed up by type erasure.

Reply:
It won't if I explicitly call `A` a `TypeTag`